services/minespace-web/src/components/Forms/projects/informationRequirementsTable/IRTFileImport.tsx

```diff
@@ -77,21 +77,21 @@ export const IRTFileImport: FC<IRTFileImportProps> = ({
     renderTextColumn("create_user", "Imported By"),
   ];
 
-  const handleCreateInformationRequirementsTable = (
+  const handleCreateInformationRequirementsTable = async (
     projectGuid: string,
     file: IFileInfo,
     documentGuid: string
   ) => {
-    dispatch(createInformationRequirementsTable(projectGuid, file, documentGuid));
+    return dispatch(createInformationRequirementsTable(projectGuid, file, documentGuid));
```

Comment:
dispatch returns an action, doesn't it? It looks like it maybe wants to be await, but hard to tell in JS files.

Reply:
I thought so too, but I guess not.  ESLINT threw an error if I added an await there. (and it works without)

Reply:
I guess the function consumer probably just wants something that _returns_? I find it odd but 🤷‍♀️ 
(I had to double-check, it does return an action)

Reply:
Yeah, I did the same thing.  🤷🏻 

Honestly the setup here isn't great, but I think there are a lot of threads to pull here 😬